cluster/addons/volumesnapshots/rbac/rbac-volume-snapshot-controller.yaml

```diff
@@ -0,0 +1,99 @@
+# RBAC file for the volume snapshot controller.
```

Comment:
I would put the rbac in the same folder as snapshot controller manifest

Reply:
ok